### DISCUSSION TITLE
Embedded Image URI Not Transformed Properly

### DISCUSSION THREAD

**Discussion by acavella** [ObsidianPublisher/obsidian-github-publisher#78] [Q&A — Plugin] [answered]
I'm having issues with image source and I can't seem to get it corrected no matte what I try.  

I'm using this to publish posts to my Jekyll blog on Github/Netlify. When publishing images, it puts them in the correct folder `assets/images` , however, the img src end up incorrect with a `../` prepending the source.

Configured with a "fixed" folder configuration.
Default Folder: _posts
Attachment Folder: assets/images

Site Structure:

```
root
  ├ index.html
  ├ _posts
    ├ 20230113-first_post.md
    └ 20230103-another_post.md
  ├ assets
    └ images
      ├ image01.jpg
      └ image02.jpg
```

It certainly makes sense that it is pre-pending `../` to the link, because `_posts` and `assets` are on the same level.  However, when the site is built/rendered, posts actually displayed from the `index.md` root level.  Is there a way to remove the `../` but continue to upload images to the `assets/images` directory?

**Comment by Mara-Li** [comment, marked as answer]
You can use the regex replacement to remove `../`
Note : you need to use the option to have the regex replacement after all other conversion.

**Reply by acavella** [reply]
Sorry I feel like I'm missing something key here or just being dense. I've set this match `../assets` and replaces it with `/assets`, but it does not seem to be performing any replacement.

![image](https://user-images.githubusercontent.com/9434191/212473385-9ed1b88a-e8ad-43b1-9a39-da6dcadc83be.png)

**Comment by Mara-Li** [comment]
you need to escape the point 
If you need check https://regex101.com/

**Reply by acavella** [reply]
Duh! Thank you, works perfectly!
